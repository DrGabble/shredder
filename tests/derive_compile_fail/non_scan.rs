#[macro_use]
extern crate shredder;

struct NotScan {}

#[derive(Scan)]
struct Test0 {
    field: NotScan
}

fn main() {}
