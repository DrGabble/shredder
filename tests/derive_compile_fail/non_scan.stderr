error[E0277]: the trait bound `NotScan: shredder::Scan` is not satisfied
 --> $DIR/non_scan.rs:6:10
  |
6 | #[derive(Scan)]
  |          ^^^^ expected an implementor of trait `shredder::Scan`
  |
  = note: required because of the requirements on the impl of `shredder::Scan` for `NotScan`
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
